traits/observers/parsing.py

```diff
@@ -0,0 +1,262 @@
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import contextlib
+from functools import reduce
+import operator
+
+import traits.observers.expression as _expr_module
+from traits.observers._generated_parser import (
+    Lark_StandAlone as _Lark_StandAlone
```

Comment:
Oh my, that's horrible. Camel_CaseWithUnderscores? Not much we can do about that, I guess.

Reply:
Yep, the name in the generated parser is out of our control. I suppose we could change the alias name...

Reply:
I'm curious: why are we aliasing this in the first place?

Reply:
Because this `parsing` is a public module, and I got overly excited about hiding variables from TAB auto-completion... except I failed to do this 100% because the standard library imports aren't given private names.

Reply:
Okay. We don't have any policy in place about doing this in general; I think that given that we rarely ever do a `from ... import *`, it's probably not needed.